src/components/Location/LocationSheet.tsx

```diff
@@ -204,6 +221,25 @@ export function LocationSheet({
     },
   });
 
+  const deleteAssociation = useMutation({
+    mutationFn: ({ status, location, id }: LocationAssociationUpdate) => {
+      if (status === "active") {
+        throw new Error(t("location_cancellation_failed"));
+      }
```

Comment:
let's not even allow `deleteAssociation` to be invoked in the first place when it's active instead.

```suggestion
```